<launch>
  
    <arg name="yaml" default="kitting_manager"/>

    <rosparam command="load" file="$(find assembly_kitting_manager)/param/$(arg yaml).yaml" ns="kitting_manager" />
    <node name="kitting_manager" pkg="assembly_kitting_manager" type="kitting_manager.py" output="screen"/>


</launch>